src/components/apiDelegation/offered/ChooseApiPage/ChooseApiPage.module.css

```diff
@@ -12,10 +12,25 @@
     margin-bottom: 30px;
     width: 100%;
   }
+  .searchSection{
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+  }
+
+
+  .filter {
+    padding-top: 0.5rem;
+  }
 }
 
 @media only screen and (min-width: 769px) {
   .searchSection{
+    display: flex;
+    align-items: center;
+  }
+
+  .searchField{
```

Comment:
Should we have some bottom on pc view? The picture is with the bottom-margin.
<img width="1030" alt="image" src="https://github.com/Altinn/altinn-access-management-frontend/assets/31471142/a2287903-7118-4883-b6c1-4b2682d79ca9">


```suggestion
  .searchSection{
  margin-bottom: 50px;
```